mungegithub/pulls/label_based_on_path.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pulls
+
+import (
+	"fmt"
+	"strings"
+
+	github_util "k8s.io/contrib/github"
+	"k8s.io/contrib/mungegithub/config"
+	"k8s.io/kubernetes/pkg/util"
+
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+	"github.com/spf13/cobra"
+)
+
+var (
+	_        = fmt.Print
```

Comment:
It makes it so I can add/remove `fmt.Print` lines to debug without the compiler whining the "fmt" is unused.